docs/custom_models.md

```diff
@@ -0,0 +1,104 @@
+# Custom Models
+
+HELM comes with more than a hundred built-in models. If you want to run a HELM evaluation on a model that is not built-in, you can configure HELM to add your own model. This also allows you to evaluate private models that not publicly accessible, such as a model checkpoint on local disk, or a model server on a private network
+
+HELM comes with many built-in `Client` classes (i.e. model API clients) and `Tokenizer` clients. If there is already an existing `Client` and `Tokenizer` class for your use case, you can simply add it to your local configuration.
+
+If you wish to evaluate a model not covered by an existing `Client` and `Tokenizer`, you can implement your own `Client` and `Tokenizer` subclasses. Instructions for adding custom `Client` and `Tokenizer` subclasses will be added to the documentation in the future.
+
+## Creating a model deployments configuration
+
+To create a local model deployments configuration file, determine the location of your local configuration folder. If you are using the `--local-path` flag with `helm-run`, the specified folder of the flag is the local configuration folder. Otherwise, the local configuration folder is `./prod_env/` under your current working directory by default.
+
+Create a file called `model_deployments.yaml` underneath that directory (e.g. `./prod_env/model_deployments.yaml`).
```

Comment:
Maybe call this `custom_model_deployments.yaml` or something, since `model_deployements.yaml` is already a filename, just to avoid a clash.

Reply:
This requires a code change, so I'll defer it.